package.json

```diff
@@ -63,14 +63,14 @@
     "@skyux/indicators": "4.7.1",
     "@skyux/inline-form": "4.1.0",
     "@skyux/layout": "4.4.3",
-    "@skyux/lists": "4.5.1",
-    "@skyux/lookup": "4.8.3",
+    "@skyux/lists": "4.6.0",
+    "@skyux/lookup": "4.9.0",
     "@skyux/modals": "4.5.1",
     "@skyux/omnibar-interop": "4.0.1",
     "@skyux/popovers": "4.4.0",
     "@skyux/router": "4.0.1",
-    "@skyux/tabs": "4.6.1",
-    "@skyux/theme": "4.13.2",
```

Comment:
Just a note while I was in here looking at something. Make sure that on all of these we also update the `peerDependencies` to include 

```
"@skyux/theme": "^4.14.0"
```